bottles/backend/managers/manager.py

```diff
@@ -345,17 +345,16 @@ def check_runners(self, install_latest: bool = True) -> bool:
 
         # lock winemenubuilder.exe
         for runner in runners:
-            winemenubuilder_paths = [
-                f"{runner}lib64/wine/x86_64-windows/winemenubuilder.exe",
-                f"{runner}lib/wine/x86_64-windows/winemenubuilder.exe",
-                f"{runner}lib32/wine/i386-windows/winemenubuilder.exe",
-                f"{runner}lib/wine/i386-windows/winemenubuilder.exe",
-            ]
-            for winemenubuilder in winemenubuilder_paths:
-                if winemenubuilder.startswith("Proton"):
-                    continue
-                if os.path.isfile(winemenubuilder):
-                    os.rename(winemenubuilder, f"{winemenubuilder}.lock")
+            if runner in self.supported_proton_runners:
```

Comment:
```suggestion
            if runner not in self.supported_proton_runners:
```
If I'm not mistaken, we add this lock only if it's *not* a Proton runner